pom.xml

```diff
@@ -223,6 +223,14 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-inline</artifactId>
+			<version>${mockito.version}</version>
+			<type>jar</type>
+			<scope>test</scope>
+		</dependency>
```

Comment:
We try to push against using `Mockito / ...` based mocks as these come with a high maintenance cost. I know there are still some parts of code that use this approach, but it would be great to avoid it for the new code.

Adding new extended capabilities to `Mockito` seems to be heading in the opposite direction.